joomla/joomla-cms
# [4.3][Guided Tours] Remove unnecessary title generation on copy or duplicate

Pull Request for Issue #40024.

### Summary of Changes

The tour and step titles get a number when duplicated. 
However, this is unnecessary, especially that it breaks translations if the title is a language key. 

### Testing Instructions

Go to the tours list view.
Edit a tour.
Save as Copy. 
You should end up with a disabled tour but matching the one it was copied from.

Select one or several tours. 
Go to 'Actions' and select 'Duplicate'.
You should end up with disabled tour(s) but matching the one(s) it(they) was(were) copied from.

Go to the step view of a tour (to access the step view, click on the number next to the tour in tours list view).
Edit a step.
Save as Copy.
You should end up with a disabled step but matching the one it was copied from.

### Actual result BEFORE applying this Pull Request

A tour title could end up looking like:
COM_GUIDEDTOURS_TOUR_USERS_TITLE (2)
in tours list view because it can no longer be translated (if the original title is a language key).

### Expected result AFTER applying this Pull Request

You get the proper translation for the tour if the title is a language key.
Titles no longer get an additional differentiable number.

### Link to documentations
Please select:

- [ x] No documentation changes for manual.joomla.org needed
